src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const targetMaxRadius = 2000
```

Comment:
What do you think about naming constants like `TARGET_MAX_RADIUS` ?

Reply:
Yeah seems better